doctrinal-works-v2/original/10-chapter-10.adoc

```diff
@@ -1289,9 +1293,12 @@ but Christ liveth in me; and the life that I now live in the flesh,
 I live by the faith of the Son of God, who loved me, and gave himself for me.`' +++[+++Mark,]
 This should be every christian`'s life and living.
 
+[.signed-section-signature]
 G+++.+++ F.
 
+[.signed-section-context-close]
 Worcester-Prison, the 9th day of the 11th month, 1676.
 
+[.signed-section-context-close]
```

Comment:
Idk if this is the right thing to do, but it didn't seem right as a postscript because it seems to be referring to him being in jail in the SSCC above it? idk!

Reply:
yeah, it doesn't fit super good, but i think you made the right call 👍